app/scripts/modules/core/src/task/monitor/MultiTaskMonitor.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { TaskMonitor } from './TaskMonitor';
+import { TaskMonitorStatus } from 'core/task/monitor/TaskMonitorStatus';
+import { TaskMonitorError } from 'core/task/monitor/TaskMonitorError';
+
+import './multiTaskMonitor.component.less';
+
+interface IMultiTaskMonitorProps {
+  monitors: TaskMonitor[];
+  title: string;
+  closeModal: () => void;
+}
+
+export const MultiTaskMonitor = ({ monitors = [], title, closeModal }: IMultiTaskMonitorProps) => {
+  const [hasErrors, setHasErrors] = React.useState(monitors.some(monitor => monitor.error));
+  const isVisible = monitors.some(monitor => monitor.submitting || monitor.error);
+  const clearErrors = (): void => {
+    monitors.forEach(monitor => (monitor.error = null));
+    setHasErrors(false);
+  };
+
+  if (!isVisible) {
+    return null;
+  }
+
+  return (
+    <div className="overlay overlay-modal multi-task-monitor">
+      <div className="modal-header">
```

Comment:
same question about `<Modal.Header/>`, etc

Reply:
yeah, for sure, was copy-paste from angular template